src/main/java/com/ably/kafka/connect/utils/RecordHeaderConversions.java

```diff
@@ -42,11 +42,14 @@ public static MessageExtras toMessageExtras(final SinkRecord record) {
         JsonUtils.JsonUtilsObject extrasObject = null;
 
         Object pushExtras = null;
-        final byte[] key = (byte[]) record.key();
-        if (key != null) {
+        final Object recordKey = record.key();
+        if (recordKey != null) {
             kafkaExtras = JsonUtils.object();
-            final String keyValue = Base64.getEncoder().encodeToString(key);
-            kafkaExtras.add("key", keyValue);
+            if (recordKey instanceof byte[]){
```

Comment:
Same as above re spacing around `{` and `}`

Reply:
This will hopefully have disappeared by 55b4ca1e0a6f0f2fd05ae2d3cace91f18dfc2cfb